website/docs/addon_maya_xgen_artist.md

```diff
@@ -11,96 +11,181 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+![Xgen Create Publish](assets/maya/artist/xgen_create_2.gif)
 
-## Create
+:::info
+Use your character **model or rig** product loaded via **AYON Loader** before starting to setup **Xgen**
+::: 
 
-Create an Xgen instance to publish. This needs to contain only **one Xgen collection**.
+Once your `Xgen` setup is ready, select it and create an `Xgen` publish instance type. This needs to contain only **one Xgen collection**.
 
 `AYON > Create... > Xgen`
 
-You can create multiple Xgen instances if you have multiple collections to publish.
+![Xgen Create Publish](assets/maya/artist/xgen_create.gif)
 
-### Publish
+:::tip
+You can create multiple `Xgen` publish instances within single workfile each representing single `Xgen` collection. You can pick different name `Variant` for such occasions resulting in `XgenHead` and `XgenBody` etc. But most of the times single collection will do as it can consist of multiple descriptions representing these anyway.
+:::
 
-The publishing process will grab geometry used for Xgen along with any external files used in the collection's descriptions. This creates an isolated Maya file with just the Xgen collection's dependencies, so you can use any nested geometry when creating the Xgen description. An Xgen version will consist of:
+### Publishing Xgen setup
+
+Once you are ready with your `Xgen` setup you can trigger `Publish` action via `AYON`>`Publish`.
+
+![Xgen Publish](assets/maya/artist/xgen_publish.gif)
+
+The publishing process will grab geometry used for grooming `Xgen` along with any external files used in the collection's descriptions. This creates an isolated Maya file with just the `Xgen` collection's dependencies, so you can use any nested geometry when creating the `Xgen` description. An `Xgen` version will consist of:
 
 - Maya file (`.ma`) - this contains the geometry and the connections to the Xgen collection and descriptions.
 - Xgen file (`.xgen`) - this contains the Xgen collection and description.
 - Resource files (`.ptx`, `.xuv`) - this contains Xgen side car files used in the collection and descriptions.
 
-## Load
+You should end up with new `Xgen` product `v001` in your project and such a product can be later loaded to any maya workfile within a project...
+
+### Loading Xgen
 
-Open the Loader tool, `AYON > Loader...`, and navigate to the published Xgen version. On right-click you'll get the option `Reference Xgen (ma)`
-When loading an Xgen version the following happens:
+Open the `AYON > Loader...`, and navigate to the published `Xgen` version. On right-click action you'll get the option to `Reference Xgen (ma)` into your active workfile.
+
+![Xgen Load](assets/maya/artist/xgen_load.gif)
+
+:::warning
+Save your maya workfile if you didn't before attempting to load Xgen product into your scene as it needs to be saved first because of Xgen side car files writting! Otherwise warning message appears telling you to do so...
```

Comment:
```suggestion
Save your maya workfile if you didn't before attempting to load Xgen product into your scene as it needs to be saved first because of Xgen side car files writing! Otherwise warning message appears telling you to do so...
```